docs/_deploying/heroku.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Deploying to Heroku
+order: 0
+slug: heroku
+lang: en
+layout: tutorial
+permalink: /deploying/heroku
+redirect_from:
+  - /deploying
+---
+# Deploying to Heroku
+
+<div class="section-content">
+This guide will walk you through preparing and deploying a Slack app using [Bolt for JavaScript](/bolt-js) and the [Heroku platform](https://heroku.com/). Along the way, we’ll download a Bolt Slack app, prepare it for Heroku, and deploy it.
+</div>
+
+When you’re finished, you’ll have this ⚡️[Getting Started with Heroku app](https://github.com/mwbrooks/bolt-js-getting-started-with-heroku-app) to run, modify, and make your own.
+
+---
+
+### Get a Bolt Slack app
+
+We'll continue from [Getting Started with Bolt for JavaScript][getting-started-guide] to learn how to prepare and deploy a Slack app to Heroku.
+
+Download the Getting Started app and change into its directory:
+
+```shell
+git clone https://github.com/mwbrooks/bolt-js-getting-started-app.git
+cd bolt-js-getting-started-app/
+```
+
+> 💡 You also can follow along with an existing Bolt app and we'll mention any extra steps that are needed.
+
+Now that you have the app, let's prepare it for Heroku.
+
+---
+
+### Prepare the app for Heroku
+
+Heroku is a flexible platform that uses a few conventions to understand how to host your app. In this section, we'll update your app to use those conventions.
+
+**1. Use a Git repository**
+
+Before you can deploy your app to Heroku, you need to [install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), initialize a local Git repository, and commit your code to it.
+
+> 💡 Skip this step if you used `git clone` in the previous section because you already have a Git repository.
+
+To initialize a local Git repository and run the command:
+
+```shell
+git init
+```
+
+Now add your code and commit it to the repository:
+
+```shell
+git add .
+git commit -m "Initial commit"
+```
+
+**2. Add a Procfile**
+
+Every Heroku app uses a special file called `Procfile` that tells Heroku how to start your app. A Bolt Slack app will be started as a web server with a public web address.
+
+Create a new file called `Procfile` (without any extension) in your app's root directory with the following content:
+
+```yaml
+web: node app.js
+```
+
+Once you've saved the file, let's commit it to your Git repository:
+
+```shell
+git add Procfile
+git commit -m "Add Procfile"
+```
+
+**3. Extra steps for existing apps**
```

Comment:
Instead of a new section, I'd use a 💡 callout like you did above.

Reply:
I really like this suggestion. It helps to tighten up this lengthy section. Fixed in 7ee5c1d